Newsletters/Yearn Finance Newsletter #45/Yearn Finance Newsletter #45.md

```diff
@@ -1,66 +1,66 @@
 
-Check out the vaults here at [yearn.finance/vaults](https://yearn.finance/vaults).
+Echa un vistazo a las vaults aquí en [yearn.finance/vaults](https://yearn.finance/vaults).
 
-## **How To Swap Into Yearn Vaults**
+## **Como hacer SWAP para entrar a las vaults de Yearn**
 
 ![](image3.jfif)
 
-One of the best ways to do Yearn Vaults swaps funds from one vault to another is typically using [curve.fi](https://curve.fi/) or with zaps built into Yearn's UI. This is gas efficient, but may cost extra in slippage or additional fees.
+Una de las mejores maneras de hacer SWAP de una vault a otra es utilizando [curve.fi](https://curve.fi/) o con la opcion de ZAPS disponible en la interfaz de Yearn. Esto es eficiente con el gas pero puede causar un deslizamiento adicional o comisiones adicionales.
 
-Another way of migrating funds from Compound and AAVE to Yearn is with the use of [bowswap.finance](https://bowswap.finance/). It lets you withdraw from the protocol and deposit into a Yearn vault, with it additionally showing both protocol's APYs for comparison and allowing you to select multiple tokens at a time. No extra fees!
+Otra forma de migrar fondos desde Compund y Aave a Yearn es utilizando [bowswap.finance](https://bowswap.finance/). Esto te permite retirar del protocolo y depositar en una vault de Yearn, mostrando a su vez el APY de cada protocolo para poder hacer una comparativa y permitiendo seleccionar múltiples tokens al mismo tiempo. ¡Sin comisiones extras!
 
-Thanks to [cryptouf](https://twitter.com/cryptouf) for curating this short guide.
+Gracias a [cryptouf](https://twitter.com/cryptouf) por hacer la curación de esta pequeña guía.
 
-## **Cumulative Protocol Revenue Reaches $39M**
+## **Los ingresos acumulados del protocolo alcanzan los 39M de dólares**
 
 ![](image4.png)
 
-Yearn has earned over $39 million in revenue, since launch (excluding the interest earned by depositors). In the past 30 days, over $5 million dollars in revenue has been generated by the protocol. An extrapolation of this data projects annual revenue to be nearly $30 million.
+Yearn ha generado ingresos de casi 39 millones de dólares desde su lanzamiento (excluyendo los intereses generados por los depositantes). En los últimos 30 días, casi 5 millones de dolares han sido generados en ingresos del protocolo. Esta data permite proyectar ingresos anuales de casi 30 millones de dolares.
```

Comment:
AS-IS: Esta data permite proyectar ingresos anuales de casi 30 millones de dolares.
TO-BE: Estos datos permiten proyectar ingresos anuales de casi 30 millones de dólares.